### PR TITLE
enhance dao UI

### PR DESCRIPTION
**Pull request by 0oM4R**
### Description

some changes in the Dao page UI

### Changes

- add section names 
- improve spacing 
- replace loading progress with skeleton

### Related Issues

- #553 

### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [x] Code format and docstrings

### CODE REVIEW

**Comment by zaelgohary** [comment]
I tried voting for a proposal & the percentage got a bit off.

image

File: src/portal/views/Dao.vue [hunk 1]
```diff
@@ -1,199 +1,215 @@
...
+                  </v-card-subtitle>
+                  <v-card-text class="pb-0">
+                    Description:
+                    <div class="pl-2">
+                      <p class="text--secondary">
+                        {{ proposal.description }}
+                        <a v-bind:href="proposal.link" v-bind:target="'blank'">More details</a>
+                      </p>
+                    </div>
+                    <p v-if="proposal.end > Date.now()">
```
Comment (zaelgohary) [inline src/portal/views/Dao.vue]: Can we change the style for the proposal hash & description to be like this?

File: src/portal/views/Dao.vue [hunk 2]
```diff
@@ -1,199 +1,215 @@
...
+                    {{ proposal.action.toUpperCase() }}
+                  </v-card-title>
+                  <v-card-subtitle class="pb-0">
+                    Proposal hash:
+                    <div class="pl-2">
+                      <p class="text--secondary">{{ proposal.hash }}</p>
+                    </div>
+                  </v-card-subtitle>
+                  <v-card-text class="pb-0">
+                    Description:
```
Comment (zaelgohary) [inline src/portal/views/Dao.vue]: Make all the titles bold.

**Review by zaelgohary — CHANGES_REQUESTED** [review]

File: src/portal/views/Dao.vue (hunk 1 above)
Comment (0oM4R) [reply src/portal/views/Dao.vue]: on the same line, you mean?

**Comment by 0oM4R** [comment]
> I tried voting for a proposal & the percentage got a bit off.
@zaelgohary 
Screenshot from 2023-04-04 12-01-20
Screenshot from 2023-04-04 12-01-02
the reason is the vote weight, we can remove the smallest percentage to avoid unclear text on the bar

**Comment by zaelgohary** [comment]
@0oM4R Maybe we can add a note that says the voting percentage depends on the weight of each farm?

**Comment by 0oM4R** [comment]
> @0oM4R Maybe we can add a note that says the voting percentage depends on the weight of each farm?

I think we need to have docs for it first then we can mention this point and ref the docs link, we should open new issue

**Review by zaelgohary — APPROVED** [review]
LGTM

**Pull request merged**